subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -1259,6 +1277,18 @@ int nrf_cloud_pgps_process_update(uint8_t *buf, size_t len)
 	size_t need;
 	int64_t gps_sec;
 
+	/* Note:
+	 * When using the HTTP transport, any error returned by this
```

Comment:
is this just saying that for the HTTP transport option, nrf_cloud_pgps_finish_update() is called "automatically" for the three cases (transport error, function error, successful completion)?
and for the custom transport option, the application needs to do it?

Reply:
yes

Reply:
Could this be simplified a bit?

```
	 * When using a custom transport, the application must call
	 * nrf_cloud_pgps_finish_update() for: transport errors,
	 * an error return from this function, or successful completion of
	 * the download.
	 * When using the HTTP transport, nrf_cloud_pgps_finish_update()
	 * is called by download_client_callback().
```

Reply:
yeah, sorry -- I have a similar rewording I'll commit shortly.